config/crd/bases/infrastructure.cluster.x-k8s.io_azuremanagedcontrolplanes.yaml

```diff
@@ -18,6 +18,19 @@ spec:
     singular: azuremanagedcontrolplane
   scope: Namespaced
   versions:
+  - additionalPrinterColumns:
```

Comment:
this should be auto generated by adding kubebuilder annotations on the type definitions


See https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/api/v1beta1/azurecluster_types.go#L80 for an example

Reply:
The file edited here is actually generated, so we shouldn't make changes to it.

Instead, add a [kubebuilder annotation](https://book.kubebuilder.io/reference/markers.html) similar to what @CecileRobertMichon linked to above, and then run `make generate`.

Reply:
done!, could you please a look